packages/core/medusa-test-utils/src/medusa-test-runner-utils/bootstrap-app.ts

```diff
@@ -0,0 +1,83 @@
+import express from "express"
```

Comment:
It would be nice if this and the init DB process was a bit simpler. Nothing to do for now, just talking out loud 😄

Reply:
yes I do agree, but probably another time in another pr, it would also be nice that those utils can be used by any users. As of now, they could probably use the medusa runner in their project, but the module runner should not care about the orm and probably just use knex for a util if you still need some sort of interaction level like we do for the medusa runner. But lets keep that for another time, I have some ideas but it is not yet a prio